tasks/collectors/run-collectors/run-collectors.yaml

```diff
@@ -52,15 +60,19 @@ spec:
                 ARGS=("${ARGS[@]}" --"${arg_name}" "${arg_value}")
             done
             # Execute collector
-            timeout "$TIMEOUT" python3 "$type" "${ARGS[@]}" | tee "$RESULTS_DIR"/"$2"-"$name".json
+            # hard coding tenant now
+            mode="tenant"
```

Comment:
I thought this PR was just making the repo configurable. What is this change?

Reply:
this comment is about outdated code. But a subsequent commit adds the missing `collector_type` argument.

Reply:
The PR is titled `make collectors repository configurable` shouldn't all the other commits have been part of a separate PR?

Reply:
They are all needed to make the feature work. I can squash them now if you prefer.

Reply:
Yeah, my preference would be to squash them and make the commit say something generic like `various improvements for executing collectors` or something. Bonus if you give a short summary of what was done in the commit body

Reply:
squashed and description updated.